msu/utilities/array.nut

```diff
@@ -0,0 +1,55 @@
+::MSU.Array <- {
+	function getRandom( _array, _start = 0, _end = null )
```

Comment:
I believe Taro had the (imo very good) suggestion to rename this to rand(), which is shorter.

Reply:
Sure